docs-source/content/userguide/managing-operators/installation/_index.md

```diff
@@ -109,7 +109,9 @@ Then install the 3.x operator using the [installation](#install-the-operator-hel
 The following instructions will be applicable to upgrade operators within the 3.x release family
 as additional versions are released.
 
-To upgrade the operator, use the `helm upgrade` command. When upgrading the operator,
+To upgrade the operator, use the `helm upgrade` command. Make sure that the
+`weblogic-kubernetes-operator` repository in your local machine is at the
```

Comment:
in your local machine -> on your local machine

Reply:
updated. thanks!